### PR TITLE
style: improve responsiveness of roadmap heading

### PR DESCRIPTION
**Pull request by ShaneMaglangit**
**What does this PR do?**

Fixes `roadmap-heading`'s weird alignment on <767 screen width by removing `justify-content-center`. It doesn't really need this attribute because both the heading and paragraph are always in their own row and must be left aligned at all times.

**Before**

image

**After**

image

### CODE REVIEW

**Comment by MichaelSolati** [comment]
@ShaneMaglangit is there a GitHub issue this PR is for?

**Comment by ShaneMaglangit** [comment]
@MichaelSolati Hey! I don't think there is any. I just randomly came across it while browsing the site.

I figured that it was a very simple fix and opted to send a PR directly without raising an issue for it.

**Comment by MichaelSolati** [comment]
Please create an issue on amplication/amplication and wait for approval/validation of the issue. You can tag me once you create the issue.

**Review by MichaelSolati — APPROVED** [review]

**Pull request merged**
